lib/rbvmomi/logging.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module RbVmomi
+  class << self
+    attr_writer :logger
+  end
+
+  def self.logger
+    require 'logger'
+    @logger ||= Logger.new($stderr)
```

Comment:
```suggestion
    @logger ||= begin
      require 'logger'
      Logger.new($stderr)
    end
```

Minor, but this prevents it from checking the require over and over.  Also, if someone happens to set a custom logger ahead of time then it will never try to require logger for no reason.